tests/base.py

```diff
@@ -72,7 +72,7 @@ def configure_global_variables(self):
         self.command('let g:taskwiki_taskrc_location="{0}"'.format(self.taskrc_path))
         self.command('let g:vimwiki_list = [{"syntax": "%s", "ext": ".txt","path": "%s"}]' % (self.markup, self.dir))
 
-    def setup(self):
+    def setup_method(self):
```

Comment:
This seems unrelated to the tasklib fix. What's the motivation, is it necessary as well?

Reply:
I got a [deprecation warning from pytest](https://docs.pytest.org/en/7.2.x/deprecations.html#setup-teardown).

Reply:
Oh, okay. That should've been a separate commit then, with its own commit message saying so. Could've saved us the confusion 😉

Reply:
I set out to fix the tests, and fixed an error or warning I could. Do you want me to create another PR?

Reply:
No, not another PR, just saying that if this had been a separate commit in this PR with a commit message that sheds some light on why this is being done I wouldn't have needed to ask this question.